l10n_ec_ote/README.rst

```diff
@@ -0,0 +1,107 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Maybe it's better to use the new methodology and let the OCA bot create the readme based on the files. For example: https://github.com/OCA/project/tree/14.0/project_category/readme